src/main/java/net/dv8tion/jda/core/requests/restaction/MessageAction.java

```diff
@@ -436,6 +440,7 @@ public MessageAction addFile(final File file)
 
     /**
      * Adds the provided {@link java.io.File File} as file data.
+     * <br><u>The file stream will be closed upon execution!</u>
```

Comment:
Would rephrase this similar to the suggestion of class definition ("this method creates a stream for reading the file, which should be cleaned up via clearFiles()")